awx/main/redact.py

```diff
@@ -8,7 +8,7 @@
 
 class UriCleaner(object):
     REPLACE_STR = REPLACE_STR
-    SENSITIVE_URI_PATTERN = re.compile(r'(\w+:(\/?\/?)[^\s]+)', re.MULTILINE)  # NOQA
+    SENSITIVE_URI_PATTERN = re.compile(r'((http|https|ssh):(\/?\/?)[^\s]+)', re.MULTILINE)  # NOQA
```

Comment:
`\w+` is too greedy for large strings that don't contain URLs

Reply:
Can this also be `git:`?

Reply:
https://www.git-scm.com/docs/git-clone#_git_urls_a_id_urls_a

From what I can tell, the only transports that actually allow username/pass as part of the netloc are http/s and ssh.

Reply:
Would _accidentally_ using a username and password in a `git` netloc be a common enough mistake? If so, would not including here prevent it from being redacted from logs, etc.?

Reply:
Yea, that's a fair point - I guess somebody could put in something that *doesn't* work, like:

`git://user:pass@host`

I'll add it.